me-md/me-md
# Add navigation for existing screens

## What does this PR do?

- [x] Adds navigation for existing screens
- [x] Styles and cleans up existing screens

### Where should the reviewer start?

- Notice how navigation works in navigation/AppContainer/AppContainer.js
- Look at onPress functions and how navigation prop is used in screens

#### How should this be manually tested?

- `npm start` and run on IOS and Android device to double check styling and navigation looks proper

#### Have tests been written for all functionality?

- No

#### What are the relevant tickets?

#25 #26 #27 #28 

#### Screenshots (if appropriate)

<img width="403" alt="Screen Shot 2019-12-22 at 9 28 11 AM" src="https://user-images.githubusercontent.com/25589695/71323089-70411400-249d-11ea-9acf-edfbbd1c5f53.png">
<img width="403" alt="Screen Shot 2019-12-22 at 9 28 14 AM" src="https://user-images.githubusercontent.com/25589695/71323090-70411400-249d-11ea-8442-e09dd265b8bb.png">
<img width="403" alt="Screen Shot 2019-12-22 at 9 28 20 AM" src="https://user-images.githubusercontent.com/25589695/71323095-746d3180-249d-11ea-8e8f-24540a41700c.png">
<img width="403" alt="Screen Shot 2019-12-22 at 9 28 27 AM" src="https://user-images.githubusercontent.com/25589695/71323096-746d3180-249d-11ea-9abc-84f29d388640.png">
